docs/en/modules/ws2812.md

```diff
@@ -15,14 +15,17 @@ through the serial port (it will be reconfigured to support WS2812-like
 protocol). If you want to keep access to Lua's console, you will have to 
 use an other input channel like a TCP server (see [example](https://github.com/nodemcu/nodemcu-firmware/blob/master/examples/telnet.lua))
 
-## ws2812.init(mode)
+## ws2812.init()
 Initialize UART1 and GPIO2, should be called once and before write().
 Initialize UART0 (TXD0) too if `ws2812.MODE_DUAL` is set.
 
+#### Syntax
+`ws2812.init([mode])`
+
 #### Parameters
-- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted) or `ws2812.MODE_DUAL`.
-In `ws2812.MODE_DUAL` mode you will be able to handle two strips in parallel but will lose access
-to Lua's serial console as it shares the same UART and PIN.
+- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted) or `ws2812.MODE_DUAL`
+
+In `ws2812.MODE_DUAL` mode you will be able to handle two strips in parallel but will lose access to Lua's serial console as it shares the same UART and PIN.
 
 #### Returns
 `nil`
```

Comment:
The code change tells the caller of `ws2812_init()` that 0 results are returned on the stack aka `nil`. Before that, if told to print the result eg, the calling function would pick up a random value (since `ws2812_init()` didn't set a return value) and thus extract a random number of items from the stack, causing the crash.